src/plugins/orangehrmPimPlugin/config/routes.yaml

```diff
@@ -508,6 +508,14 @@ apiv2_pim_validate_employee_other_email:
   requirements:
     empNumber: '\d+'
 
+apiv2_pim_my_info:
+  path: /api/v2/pim/my-info
```

Comment:
can go for `/api/v2/pim/myself` or `/api/v2/pim/me` :thinking: 
ref: https://stackoverflow.com/questions/36520372/designing-uri-for-current-logged-in-user-in-rest-applications

Reply:
Changed to /api/v2/pim/myself